lib/orderbook/errors.ts

```diff
@@ -4,12 +4,14 @@ const codesPrefix = errorCodesPrefix.ORDERBOOK;
 const errorCodes = {
   PAIR_DOES_NOT_EXIST: codesPrefix.concat('.1'),
   DUPLICATE_ORDER: codesPrefix.concat('.2'),
-  OWN_ORDER_NOT_FOUND: codesPrefix.concat('.3'),
-  CURRENCY_DOES_NOT_EXIST: codesPrefix.concat('.4'),
-  CURRENCY_CANNOT_BE_REMOVED: codesPrefix.concat('.5'),
-  CURRENCY_ALREADY_EXISTS: codesPrefix.concat('.6'),
-  PAIR_ALREADY_EXISTS: codesPrefix.concat('.7'),
-  ORDER_NOT_FOUND: codesPrefix.concat('.8'),
+  ORDER_NOT_FOUND: codesPrefix.concat('.3'),
+  OWN_ORDER_NOT_FOUND: codesPrefix.concat('.4'),
+  PEER_ORDER_NOT_FOUND: codesPrefix.concat('.5'),
```

Comment:
In #589 @erkarl and I discussed going with just a single `ORDER_NOT_FOUND` error type, with the reasoning that the caller of the method to get an order will know whether they were trying to get a peer or own order already. Would it make sense to you to stick to that approach as well? If so we can drop the peer and own order not found errors, it actually looks like `PEER_ORDER_NOT_FOUND` is not currently being used.

Reply:
Yes, I overlooked that. 

The `peerPubKey`/`localId` differentiation issue isn't very neat, but we can fix it later.